autonomous_software_pkg/src/map_publisher_2.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python3
+
+import os
+import rospy
+import tf
+import tf2_ros
+import utm
+
+from enum import Enum
+from pykml import parser
+
+from geometry_msgs.msg import Point, TransformStamped
+from visualization_msgs.msg import Marker, MarkerArray
+
+
+class PlacemarkType(Enum):
+    Contour = 1
+    Lane = 2
+    Obstacle = 3
+
+
+class MapComponent:
+    def __init__(
+        self,
+        id: int = 0,
+        type: PlacemarkType = PlacemarkType.Contour,
+        list_coordinates: list = [],
+    ):
+        self.id = id
+        self.type = type
+        self.list_coordinates = list_coordinates
+
+
+class MapPublisher:
+    def __init__(self):
+
+        rospy.init_node("map_publisher", anonymous=True)
```

Comment:
If we want to avoid node name collisions with the old map publisher, I think we need this:
```suggestion
        rospy.init_node("map_publisher_2", anonymous=True)
```

Reply:
I've replaced the old file with the new one. Do you want to keep the old one?

Reply:
Ah ok, we're good then, thx.